### PR TITLE
Facets and widgets

### PR DESCRIPTION
**Pull request by adamdaventryGov**
Added functionality for facets

### CODE REVIEW

File: src/main/resources/graphql/crdc-ctdc-private-es.graphql [hunk 1]
```diff
@@ -125,11 +130,66 @@ type HomePage {
...
+    filterParticipantCountByAnatomicalCollectionSite: [GroupCount],
+    filterParticipantCountByTissueCategory: [GroupCount],
+    filterParticipantCountBySpecimenType: [GroupCount],
+    filterParticipantCountByAssessmentTimepoint: [GroupCount],
+
+    filterParticipantCountByDataFileType: [GroupCount],
+    filterParticipantCountByDataFileFormat: [GroupCount]
+}
+
+type ArmInfo2 {
```
Comment (jonkiky) [inline src/main/resources/graphql/crdc-ctdc-private-es.graphql:180]: Can you change the ArmInfo2 into ArmInfo
Same for programArms2

**Review by jonkiky — APPROVED** [review]

File: src/main/resources/graphql/crdc-ctdc-private-es.graphql [hunk 2]
```diff
@@ -125,11 +130,66 @@ type HomePage {
...
+    participantCountByReportedGender: [GroupCount],
+    participantCountByRace: [GroupCount],
+    participantCountByEthnicity: [GroupCount],
+    participantCountByCarcinogenExposure: [GroupCount],
+    participantCountByTargetedTherapy: [GroupCount],
+    participantCountByAnatomicalCollectionSite: [GroupCount],
+    participantCountByTissueCategory: [GroupCount],
+    participantCountBySpecimenType: [GroupCount],
+    participantCountByAssessmentTimepoint: [GroupCount],
+    participantCountByDataFileType: [GroupCount],
```
Comment (jonkiky) [inline src/main/resources/graphql/crdc-ctdc-private-es.graphql:159]: Can we change  participantCountByDataFileType to dataFileCountByDataFileType, 
same for participantCountByDataFileType.

Change  participantCountByAnatomicalCollectionSite to specimenCountByAnatomicalCollectionSite,
same for   participantCountByTissueCategory and   participantCountBySpecimenType

Do not forgot related the filter ones.

File: src/main/resources/graphql/crdc-ctdc-private-es.graphql (hunk 1 above)
Comment (adamdaventryGov) [reply src/main/resources/graphql/crdc-ctdc-private-es.graphql]: This has been changed

File: src/main/resources/graphql/crdc-ctdc-private-es.graphql (hunk 2 above)
Comment (adamdaventryGov) [reply src/main/resources/graphql/crdc-ctdc-private-es.graphql]: These have been updated and work with the front end

**Review by jonkiky — APPROVED** [review]

**Pull request merged**